pkg/ingester/ingester.go

```diff
@@ -2006,7 +2030,7 @@ func (i *Ingester) sendStreamingQuerySeries(ctx context.Context, q storage.Chunk
 
 	// Ensure no error occurred while iterating the series set.
 	if err := ss.Err(); err != nil {
-		return nil, 0, err
+		return nil, 0, errors.Wrap(err, "iterating ChunkSeriesSet")
```

Comment:
[nit] I think calling it "iterating" is a bit misleading. I understand this is coming from the comment above, but in practice what's happening here is that we're "sending" not "iterating".

_Same comment applies above._

Reply:
I wasn't actually looking at the comment, I was just relating to the fact that `ss.Err()` returns any error having occurred during iteration of the `ss` iterator. I don't understand how "sending" would be more descriptive, since any error from `ss.Err()` would be from failed `ChunkSeries` iteration, not sending. A failed send should result in an error returned from `client.SendQueryStream`, no?